cmd/receiver-proxy/main.go

```diff
@@ -109,10 +92,10 @@ var flags = []cli.Flag{
 		EnvVars: []string{"CONN_PER_PEER"},
 	},
 	&cli.IntFlag{
-		Name:    "max-local-requests-per-second",
-		Value:   100,
-		Usage:   "Maximum number of unique local requests per second",
-		EnvVars: []string{"MAX_LOCAL_RPS"},
+		Name:    flagMaxUserRPS,
+		Value:   0,
+		Usage:   "Maximum number of unique user requests per second per IP (set 0 to disable)",
```

Comment:
```suggestion
		Usage:   "Maximum number of unique user requests coming to user endpoint",
```

It's not actually per IP - it's altogether